model/harvestReport.go

```diff
@@ -0,0 +1,29 @@
+package model
+
+type HarvestReport struct {
+	ID               string          `json:"id" bson:"id"`
+	URL              *string         `json:"url" bson:"url"`
+	DataType         HarvestTypeEnum `json:"dataType" bson:"dataType"`
+	HarvestError     bool            `json:"harvestError" bson:"harvestError"`
+	StartTime        string          `json:"startTime" bson:"startTime"`
+	EndTime          string          `json:"endTime" bson:"endTime"`
+	ErrorMessage     *string         `json:"errorMessage" bson:"errorMessage"`
+	ChangedCatalogs  []FdkIdAndUri   `json:"changedCatalogs" bson:"changedCatalogs"`
+	ChangedResources []FdkIdAndUri   `json:"changedResources" bson:"changedResources"`
+}
+
+type FdkIdAndUri struct {
+	FdkID string `json:"fdkId"`
```

Comment:
Når det er flere forkortelser etter hverandre kan det bli litt vanskelig å lese, men tror dette er riktig gitt at det skal være exposed fields (uppercase forbokstav).

```suggestion
type FDKIDAndURI struct {
	FDKID string `json:"fdkId"`
```

Reply:
b2f833b